src/styles/App.scss

```diff
@@ -0,0 +1,10 @@
+@import '~@blueprintjs/core/lib/css/blueprint.css';
+@import '~@blueprintjs/icons/lib/css/blueprint-icons.css';
+@import '~@blueprintjs/core/lib/css/blueprint.css';
+@import '~@blueprintjs/icons/lib/css/blueprint-icons.css';
+
+@import "variables";
+@import "common";
+@import "header";
+@import "sections";
+@import '//cdn-images.mailchimp.com/embedcode/horizontal-slim-10_7.css';
```

Comment:
This isn't super germane to this PR, but this mailchimp stylesheet feels like it should be closer to the component where we're instantiating the mailchimp embed. @experimatt are we using these anywhere outside of that embed? I do like the idea of centralizing the imports here,

Reply:
Nope, just in the embed. I can move it later, now that we've got separate scss files for each component.